MerlinAU.sh

```diff
@@ -3501,12 +3527,11 @@ _high_risk_phrases_interactive_()
             else
                 Say "*WARNING*: Found high-risk phrases in the changelog file."
                 Say "Please run script interactively to approve the firmware update."
+                Update_Custom_Settings "FW_New_Update_Changelog_Approval" "BLOCKED"
```

Comment:
Duh this needs to be added wherever the notification email is sent. I obviously missed this one. Good thing you understood the solution I was putting down!